contracts/BSCValidatorSet.sol

```diff
@@ -195,49 +195,15 @@ contract BSCValidatorSet is IBSCValidatorSet, System, IParamSubscriber, IApplica
         uint8,
         bytes calldata msgBytes
     ) external override onlyInit onlyCrossChainContract initValidatorExtraSet returns (bytes memory responsePayload) {
-        (IbcValidatorSetPackage memory validatorSetPackage, bool ok) = decodeValidatorSetSynPackage(msgBytes);
-        if (!ok) {
-            return CmnPkg.encodeCommonAckPackage(ERROR_FAIL_DECODE);
-        }
-        uint32 resCode;
-        if (validatorSetPackage.packageType == VALIDATORS_UPDATE_MESSAGE_TYPE) {
-            resCode = updateValidatorSet(validatorSetPackage.validatorSet, validatorSetPackage.voteAddrs);
-        } else if (validatorSetPackage.packageType == JAIL_MESSAGE_TYPE) {
-            if (validatorSetPackage.validatorSet.length != 1) {
-                emit failReasonWithStr("length of jail validators must be one");
-                resCode = ERROR_LEN_OF_VAL_MISMATCH;
-            } else {
-                address validator = validatorSetPackage.validatorSet[0].consensusAddress;
-                uint256 index = currentValidatorSetMap[validator];
-                if (index == 0 || currentValidatorSet[index - 1].jailed) {
-                    emit validatorEmptyJailed(validator);
-                } else {
-                    // felony will failed if the validator is the only one in the validator set
-                    bool success = _felony(validator, index - 1);
-                    if (!success) {
-                        emit validatorEmptyJailed(validator);
-                    }
-                }
-                resCode = CODE_OK;
-            }
-        } else {
-            resCode = ERROR_UNKNOWN_PACKAGE_TYPE;
-        }
-        if (resCode == CODE_OK) {
-            return new bytes(0);
-        } else {
-            return CmnPkg.encodeCommonAckPackage(resCode);
-        }
+        revert("deprecated");
```

Comment:
the name of `IbcValidatorSetPackage`and `decodeValidatorSetSynPackage` is not good now, better change to better one.

Reply:
mark the `EXPIRE_TIME_SECOND_GAP` as expire please

Reply:
fixed